gpu-operator/security.rst

```diff
@@ -0,0 +1,103 @@
+
+*****************************
+Security Considerations
+*****************************
+
+
+Pod Security Context of the Operator and Operands
+=================================================
+
+Several of the NVIDIA GPU Operator operands, such as the driver containers and container toolkit,
+require the following elevated privileges:
+
+- ``privileged: true``
+- ``hostPID: true``
+- ``hostIPC: true``
+
+The elevated privileges are required for the following reasons:
+
+- Access to the host file system and hardware devices, such as NVIDIA GPUs.
+- Restart system services such as containerd.
+- Permit users to list all GPU clients using the ``nvidia-smi`` utility.

```

Comment:
I don't think this is a valid reason:
```
Permit users to list all GPU clients using the ``nvidia-smi`` utility.
```

Let's add `Loading and unloading kernel modules` to the list